packages/aws-cdk-lib/aws-ecs/lib/cluster.ts

```diff
@@ -69,9 +69,16 @@ export interface ClusterProps {
    * If true CloudWatch Container Insights will be enabled for the cluster
    *
    * @default - Container Insights will be disabled for this cluster.
+   * @deprecated See {@link containerInsightsV2}
    */
   readonly containerInsights?: boolean;
 
+  /**
+   * The CloudWatch Container Insights configuration for the cluster
+   * @default - Container Insights will be disabled for this cluster, but this may be overridden by ECS account level settings.
```

Comment:
Slightly more readable, and offers a link to the enum value
```suggestion
   * @default ContainerInsights.DISABLED This may be overridden by ECS account level settings.
```

Reply:
👍